crates/db/src/database.rs

```diff
@@ -242,7 +242,7 @@ impl DbProvider for Database {
         let s = session::ActiveModel {
             token: Set(session_token.to_owned()),
             created: Set(created),
-            user_fk: Set(user.id as i64),
+            user_fk: Set(i64::from(user.id)),
```

Comment:
Not a fan of ``::from()` instead of `as`. I think `as` is easier to read and makes the intend of casting more clear.